### DISCUSSION TITLE
How to ocr for %?

### DISCUSSION THREAD

**Discussion by voska7** [SubtitleEdit/subtitleedit#5951] [Q&A]
i tried to expand, but it's not expending for this % character. any suggestion how can i do that?

https://i.postimg.cc/XvhtVBjq/image.png

edit:
also for this one
https://i.postimg.cc/CKhJWjWk/image.png

**Comment by Elheym** [comment]
+1.
Sometimes I've faced the same issue.

**Comment by niksedk** [comment]
It's possible but a little bit hidden...

Double click in the list view in the main OCR window on the image with `%` (or right click + inspect nocr matches for current line)
Go to the first wrong letter for the `%` sign (probably an `o` like here) in the list, and right click,  choose "Add better multi match":
![image](https://user-images.githubusercontent.com/1261331/169635716-27500ce5-1c51-47fe-86fd-3c6678069aba.png)

Set the expand count to 3, and enter `%`
![image](https://user-images.githubusercontent.com/1261331/169635772-c4d63e86-293c-4fe1-8585-45556fcca969.png)

**Comment by voska7** [comment]
is there any option to split the letter? subtitle edit suggest add "rt" instead of "r", "t"
![image](https://user-images.githubusercontent.com/42012426/170572946-84b77a2e-0576-4dde-872e-5301e167dc8a.png)
